src/connectors/siriusxm-player.ts

```diff
@@ -1,14 +1,19 @@
 export {};
 
 /*
- * On some "decade" stations SiriusXM appends the last two digits
+ * Station Meta Filter
+ * - On some "decade" stations SiriusXM appends the last two digits
  * of the release year to the song title like this: Where Is The Love (03).
+ * - On The Covers Channel, it adds Heroes (David Bowie Cover) to the track
+ * - A lot of stations seem to employ adding EXCLUSIVE to the track name, so that is now removed
  *
- * This filter removes such suffixes.
+ * Filter Station Meta
+ * - There is now a filter for much of the station meta on SiriusXM. Removes social media handles, URLs, shownames
  */
 
-const filter = MetadataFilter.createFilter({ track: removeYear });
-const removeYearRe = /\s\(\d{2}\)\s?$/g;
+const filter = MetadataFilter.createFilter({ track: removeStationMeta });
```

Comment:
How feasible is it to keep the year removal as a separate regex and remove the other information on a new pass? It's reasonable now, but I always second-guess complex regex.

Reply:
@jaccarmac That shouldn't be much issue. Honestly I'm terrible with regex and ran it through ChatGPT to optimize the 3 filters. I can definitely split them out into 3 separate filters instead.